bin/join-metadata-and-clades

```diff
@@ -106,11 +110,16 @@ def main():
 
     clades = pd.merge(clades, clades_21L, left_index=True, right_index=True, how='left')
 
+    # Add clade_legacy column as Nextstrain_clade
+    # Use yml mapping
+    clade_legacy_mapping= yaml.safe_load(open(args.clade_legacy_mapping))
+    clades["Nextstrain_clade"] = clades["clade_nextstrain"].map(clade_legacy_mapping)
```

Comment:
Great point that I wanted to address later, after getting this fix out.

I think it would be good to not require us to update the `clade-legacy-mapping.yml` for each new clade we define. Since new clades will be part of Omicron, I suggest I make the default mapping to add ` (Omicron)` as everything from now on will almost certainly be Omicron (if it isn't it's easy to overwrite by adding a line to the mapping).

Since the whole reason for this column is backwards compatibility, this won't break anything and it is also not wrong.

I'll suggest that change in a separate PR so we can merge the fix first.